vignettes/tidytranscriptomics.Rmd

```diff
@@ -422,7 +422,7 @@ Which method detects the most differentially abundant transcripts, p value adjus
 Or if we just wanted to run one method we could do that. The default method is edgeR quasi-likelihood.
 
 ```{r}
-counts_de <- counts_filtered %>%
+counts_de <- counts_scal_PCA %>%
```

Comment:
@mblue9 here I used the object we created sequentially, rather than fishing old objects. Doing this, not only we keep populating the tibble, but when we export the bibliography we get all the tools we used that far, without omitting any.

Reply:
Ok that's fine. I was doing that to show that it's not essential to run scaling and then pca before input to test_differential_abundance and that it only needs the filtered step but this makes sense.

Reply:
although if test_differential_abundance is performing scaling shouldn't it also add the scaling ref? should that be added to tidybulk?

Reply:
That's a good point. I tried to make it super clear with a long message at each execution. It is good to mention it verbally for sure.

Reply:
> although if test_differential_abundance is performing scaling shouldn't it also add the scaling ref

also good point. Good to think about that. However if you add scaling factor in the model (and not applying it directly) really what is the reference does not matter. Because there is not discretisation issues (I believe).